tests/test_model.py

```diff
@@ -16,6 +19,7 @@ def test_corrections(self):
         test_altitudes = np.linspace(0., 90., 5)
         test_azimuth = np.linspace(0., 90., 5)
         test_temperatures = np.linspace(-1., 30., 3)
+        test_wavelengths = np.linspace(320., 1100., 2)  # min, max, steps
```

Comment:
this is equivalent to 

```
test_wavelengths = np.array([320., 1100.])
```

Consider using this or make the number larger than 2.

Reply:
ooops, good catch. I put 2 for testing reasons but intended to make it larger!